webapp/components/wpt-results.html

```diff
@@ -527,15 +530,20 @@ <h3>
           }
         }
 
-        this.load(window.fetch(url, fetchOpts).then(r => {
-          if (!r.ok || r.status !== 200) {
-            return Promise.reject('Failed to fetch results data.');
+        this.load(
+          window.fetch(url, fetchOpts).then(r => {
+            if (!r.ok || r.status !== 200) {
```

Comment:
@lukebjerring Why does this throw on non-200 responses? `r.ok` means status is in [200,299], all of which should be considered successful.